packages/babel-plugin-transform-block-scoping/test/fixtures/general/closure-in-generator-or-async/exec.js

```diff
@@ -0,0 +1,36 @@
+function* a() {
```

Comment:
This test already passes with the old plugin, but it was untested (I accidentally broke it and all the tests were still passing).